core/templates/pages/android-page/android-page-root.component.spec.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the about foundation page root component.
+ */
+
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { AppConstants } from 'app.constants';
+import { PageHeadService } from 'services/page-head.service';
+
+import { MockTranslatePipe } from 'tests/unit-test-utils';
+import { AndroidPageRootComponent } from './android-page-root.component';
+
+describe('About foundation Page Root', () => {
```

Comment:
Should the name be `Android Page Root` or something similar?

Reply:
Done